lab03/sections/results.tex

```diff
@@ -1,3 +1,22 @@
 \section{Results}\label{sec:results}
-Describe the apparatus and measurement technique(s).
-Present the data.
+Fig.~\ref{fig:schematic} was realized with damping factor $\zeta = 0.1$.
+Using \eqref{eqn:damping}, $R_a = \SI{70.2}{\kilo\ohm}$.
+The opamp supply power was set at \SI{+-15}{\volt}.
+An input signal of 1 $\si{\volt}_{pp}$ at \SI{50}{\hertz} square wave was applied to the circuit.
+One pulse of the square wave will behave locally like a unit step function.
+
+The output of this circuit is shown in Fig.~\ref{fig:overshoot}.
+The large overshoot corresponds to significant underdamping, which is consistent with $\zeta = 0.1$.
```

Comment:
I think that this line might be more suitable for the discussion? Particularly since we don't have much to talk about to begin with.